timur/npmrc

```diff
@@ -0,0 +1 @@
+prefix = /app/.npm-packages
```

Comment:
:+1: This ends up being necessary for the `npm link` inside the container.  `npm link` has lots of "gotchas" unfortunately so it hasn't been as robust as I had hoped, but that reminds me... there was on change I made that helps with that.